scripts/badger/bribe_hh.py

```diff
@@ -0,0 +1,59 @@
+from decimal import Decimal
+
+from brownie import web3, interface
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r 
+
+
+def main(badger_bribe_in_aura=0, badger_bribe_in_balancer=0):
+    safe = GreatApeSafe(r.badger_wallets.treasury_ops_multisig)
+    badger_bribe_in_aura = int(badger_bribe_in_aura)
+    badger_bribe_in_balancer = int(badger_bribe_in_balancer)
+
+    badger = interface.ERC20(
+        r.treasury_tokens.BADGER, owner=safe.account
+    )
+
+    safe.take_snapshot([badger])
+
+    bribe_vault = interface.IBribeVault(
+        r.hidden_hand.bribe_vault, owner=safe.account
+    )
+
+    balancer_briber = interface.ITokenmakBribe(
+        r.hidden_hand.balancer_briber, owner=safe.account
+    )
+
+    aura_briber = interface.IAuraBribe(
+        r.hidden_hand.aura_briber, owner=safe.account
+    )
+
+    if badger_bribe_in_aura > 0:
+        prop = web3.solidityKeccak(["address"], [badger.address])
```

Comment:
in this case the proposal is built up from two variables the proposal index and the choice number which our gauge represents. [ref](https://etherscan.io/address/0x642c59937a62cf7dc92f70fd78a13cee0aa2bd9c#code#L934)

since this will change from round to round, i advise to added them also as arguments to pass in the cli

latest proposal set [tx](https://etherscan.io/tx/0xe0a89600d65a8716020c21ca099f8925d7e8c8da15945380a210e459066e1939). Proposal index is 11 and choice is 38

Reply:
checking on the graphql endpoint of snapshot. i think to leave it more programmatically solved. it's better to pass perhaps the proposal id where we want to bribe for and deduct from there the proposal index and also the choice number, since we know which choice is our target (`"80/20 BADGER/WBTC"`)